app/scripts/modules/azure/src/image/image.reader.js

```diff
@@ -28,10 +28,10 @@ module(AZURE_IMAGE_IMAGE_READER, []).factory('azureImageReader', function() {
       .withParams({ provider: 'azure' })
       .get()
       .then(
-        function(results) {
+        function (results) {
           return results && results.length ? results[0] : null;
         },
-        function() {
+        function () {
```

Comment:
🤔 Is this because of a new version of `prettier`?

Reply:
I think so. I have noticed this and some mods to one param arrow functions as well.